src/vm/readytoruninfo.h

```diff
@@ -40,6 +41,9 @@ class ReadyToRunInfo
     Crst                            m_Crst;
     PtrHashMap                      m_entryPointToMethodDescMap;
 
+    PersistentInlineTrackingMapR2R  m_persistentInlineTrackingMap;
+    PTR_PersistentInlineTrackingMapR2R m_pPersistentInlineTrackingMap;
```

Comment:
I'm personally not a big fan of this... Maybe you should make the .TryLoad function a static one that returns a pointer to a heap allocated map. Feels a bit like a hack to me